src/pages/SwapNew/components/SwapInfo/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { Trade } from '@swapr/sdk'
+
+import { AnimatePresence, motion } from 'framer-motion'
+import { useState } from 'react'
+import styled from 'styled-components'
+
+import {
+  ELEMENTS_BACKGROUND_SECONDARY,
+  ELEMENTS_BORDER_SECONDARY,
+  ELEMENTS_SPACING,
+  BorderStyle,
+} from '../../constants'
+import { SwapInfoBasics } from './SwapInfoBasics'
+import { SwapInfoDetailed } from './SwapInfoDetailed'
+
+type SwapInfoProps = {
+  loading: boolean
+  allPlatformTrades?: (Trade | undefined)[]
```

Comment:
Also here, no need for declaring `allPlatformTrades` as undefined type

Reply:
Same as above, some platforms can be `undefined` in case we're unable to fetch the details.